spec/system/hero_rotation_spec.rb

```diff
@@ -1,15 +1,26 @@
 require "rails_helper"
 
 RSpec.describe "Hero image rotation", :type => :system do
+  # it "has a different hero section background image, 5 seconds after pageload, if 1 hero image is created", js: true do
+  #   create(:page)
+  #   create(:hero_image, image: build(:alternate_image))
+  #   Capybara.current_session.driver.browser.manage.window.resize_to(1440, 1000)
+  #   visit "/"
+  #   sleep 5
+  #   expect(page).to have_xpath('//div[contains(@style,"beach-alternate.jpeg")]')
+  # end
+
   it "has a different hero section background image, 5 seconds after pageload, if 1 hero image is created", js: true do
     create(:page)
     create(:hero_image, image: build(:alternate_image))
     Capybara.current_session.driver.browser.manage.window.resize_to(1440, 1000)
     visit "/"
+    # puts page.find(:xpath, '//*[@id="page"]/div/section[1]/div[1]/div[1]')['innerHTML']
     sleep 5
+    puts page.find(:xpath, '//*[@id="page"]/div/section[1]/div[1]/div[1]')['innerHTML']
     expect(page).to have_xpath('//div[contains(@style,"beach-alternate.jpeg")]')
   end
-
+  
```

Comment:
A random line of spaces escaped! 😉

Reply:
Yes looks like a commented out section / copy of a test. Removed it.